docs/functions.rst

```diff
@@ -57,13 +57,18 @@ however functions which support this alternate API include the line:
 HIPBLAS_V2 and Deprecations
 ===========================
 
+.. _HIPBLAS_V2_DEP:
+
 As of hipBLAS version 2.0.0, hipblasDatatype_t is deprecated, along with all functions which use this type. In a future release, all uses of hipblasDatatype_t
 will be replaced by hipDataType. See the hipblasGemmEx documentation for a small exception where hipblasComputeType_t replaces hipblasDatatype_t for the
 computeType parameter.
 
-While hipblasDatatype_t is deprecated, users may use the compiler define or inline #define HIPBLAS_V2 before including the header file <hipblas.h> to
+hipblasComplex and hipblasDoubleComplex are also deprecated. In a future release, all uses of these types will be replaced with their HIP counterparts:
+hipComplex and hipDoubleComplex.
+
+While hipblasDatatype_t, ``hipComplex``, and ``hipDoubleComplex`` are deprecated, users may use the compiler define or inline #define HIPBLAS_V2 before including the header file <hipblas.h> to
```

Comment:
You mean hipblasComplex and hipblasDoubleComplex here

Reply:
Yep, thanks for the catch

Reply:
" While hipblasDatatype_t, ``hipComplex``, and ``hipDoubleComplex`` are deprecated " -- github messed my selection up